### ISSUE TITLE
Typing issue after upgrade, nothing in migration, neither in doc

### ISSUE THREAD

**Issue by Hideman85** [i18next/i18next-parser#842]
Error:
![image](https://github.com/i18next/i18next-parser/assets/16165119/94f4c196-c87f-4c8f-a3e3-af45ec47c0a3)

Expected: https://github.com/i18next/i18next-parser#jsx
```
{
  // JsxLexer default config (jsx)
  // JsxLexer can take all the options of the JavascriptLexer plus the following
  jsx: [{
    lexer: 'JsxLexer',
    attr: 'i18nKey', // Attribute for the keys
    componentFunctions: ['Trans'], // Array of components to match
  }],
}
```

Additional note: I cannot find anything named `componentFunctions` in the `dist/index.d.ts` file.

**Comment by karellm** [comment]
Can you please make a PR?

**Comment by karellm** [comment]
Should be fixed as of `8.7.0`
